kyo-core/shared/src/main/scala/kyo/Fiber.scala

```diff
@@ -23,7 +23,8 @@ object Fiber extends FiberPlatformSpecific:
 
     inline given [E, A]: Flat[Fiber[E, A]] = Flat.unsafe.bypass
 
-    private val _unit = success(())
+    private val _unit  = success(()).mask
+    private val _never = IOPromise[Nothing, Unit]().mask
```

Comment:
These constant weren't safe before since they could still be interrupted. With the new `mask` behavior, they're effectively final unless the user uses type casts.

Reply:
Ah yes, this is exactly what I was trying to communicate when I was wanting to convert these to synthetic classes. This is better though.